util/challenge/judge.py

```diff
@@ -122,9 +111,9 @@ def _get_cex_vals(deadline_dt):
     target_uts = [helpers.dt_to_ut(dt) for dt in target_dts]
     helpers.print_datetime_info("target times", target_uts)
 
-    kraken = ccxt.kraken()
-    from_dt_str = kraken.parse8601(deadline_dt.strftime("%Y-%m-%d %H:%M:00"))
-    cex_x = kraken.fetch_ohlcv("ETH/USDT", "5m", since=from_dt_str, limit=500)
+    binance = ccxt.binanceus()
+    from_dt_str = binance.parse8601(deadline_dt.strftime("%Y-%m-%d %H:%M:00"))
```

Comment:
I discussed this with Trent in a previous iteration. Here, we can not use df-py specific helpers, because it must mirror what we describe for users in the predict eth challenges.